apps/passport-server/src/services/telegramService.ts

```diff
@@ -867,22 +874,38 @@ export class TelegramService {
 
     logger(`[TELEGRAM] Starting ${bot.botInfo.username}`);
 
-    try {
-      // This will not resolve while the authBot remains running.
-      await bot.start({
-        allowed_updates: [
-          "chat_join_request",
-          "chat_member",
-          "message",
-          "callback_query"
-        ],
-        onStart: (info) => {
-          logger(`[TELEGRAM] Started bot '${info.username}' successfully!`);
+    while (retryAttempts < this.MAX_RETRY_ATTEMPTS) {
+      try {
+        // This will not resolve while the authBot remains running.
+        await bot.start({
+          allowed_updates: [
+            "chat_join_request",
+            "chat_member",
+            "message",
+            "callback_query"
+          ],
+          onStart: (info) => {
+            logger(`[TELEGRAM] Started bot '${info.username}' successfully!`);
+          }
+        });
+        return; // Exit the function if start is successful
+      } catch (e) {
+        logger(`[TELEGRAM] Error starting ${bot.botInfo.username}`, e);
+        this.rollbarService?.reportError(e);
+
+        retryAttempts++;
+        if (retryAttempts >= this.MAX_RETRY_ATTEMPTS) {
+          logger(
```

Comment:
let's make failure here more annoying for us:
- report this error to rollbar
- alert on discord
- wrap this function in a `traced` and call `setError(span, e)` in there